src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/service/NotificationService.java

```diff
@@ -47,23 +45,19 @@
  */
 @Slf4j
 @Service
-@Transactional
 @PreAuthorize("isAuthenticated()")
 public class NotificationService {
     private OffsetDateTime backupRestoreRunningSince;
     private final GlobalConfFacade globalConfFacade;
     private final TokenService tokenService;
-    private final InitializationService initializationService;
 
     /**
      * constructor
      */
     @Autowired
-    public NotificationService(GlobalConfFacade globalConfFacade, TokenService tokenService,
-            InitializationService initializationService) {
+    public NotificationService(GlobalConfFacade globalConfFacade, TokenService tokenService) {
         this.globalConfFacade = globalConfFacade;
         this.tokenService = tokenService;
-        this.initializationService = initializationService;
     }
 
     /**
```

Comment:
> If alerts are disabled, the status of all alerts true.

Is this comment up to date? Does "alerts are disabled" refer to `backupRestoreRunningSince`?

Reply:
The comment was updated.